### PR TITLE
Add azure queue storage dashboard to csp-mixin.

### PR DESCRIPTION
**Pull request by rgeyer**
This is a superset of #1334 to retain the panel reorg. I'll likely have to rebase and fix a few things after that's merged.

This *also* adds a new azure queue storage dashboard, which is a slight variation of the blob storage dashboard(s).

Finally, this also improves several things about the blob storage dashboards which were caught while developing the azure queue storage dashboard.

### CODE REVIEW

File: csp-mixin/panels/azurequeuestore.libsonnet [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    availabilityStat:
+      this.signals.azurequeuestore.availability.asStat(),
+    queueCount:
+      this.signals.azurequeuestore.queueCount.asStat()
+      + g.panel.stat.standardOptions.color.withMode('fixed')
+      + g.panel.stat.standardOptions.color.withFixedColor('purple'),
+    messageCountTotal:
+      this.signals.azurequeuestore.messageCountTotal.asStat()
+      + g.panel.stat.standardOptions.color.withMode('fixed')
+      + g.panel.stat.standardOptions.color.withFixedColor('purple'),
```
Comment (yduartep) [inline csp-mixin/panels/azurequeuestore.libsonnet:73]: We are showing counts in gray. Can you do the same?

Screenshot 2024-09-26 at 17 26 07

Screenshot 2024-09-26 at 17 26 37

File: csp-mixin/signals/azurequeuestore.libsonnet [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+      },
+
+      apiRequestErrorRate: {
+        name: 'API error rate by type',
+        description: 'Percentage of api request failure by type',
+        type: 'raw',
+        unit: 'percentunit',
+        sources: {
+          azuremonitor: {
+            expr: 'sum by (dimensionApiname) (azure_microsoft_storage_storageaccounts_queueservices_transactions_total_count{dimensionApiname!="",dimensionResponseType!="Success",%(queriesSelector)s}) / ' + s.signals.apiRequestByTypeCount.sources.azuremonitor.expr,
```
Comment (yduartep) [inline csp-mixin/signals/azurequeuestore.libsonnet:97]: love it!

File: csp-mixin/panels.libsonnet
[the remark this answers is not in the document]
Comment (rgeyer) [reply csp-mixin/panels.libsonnet:1]: Yep, I was going to suggest that on your PR, I noticed it was still there on the commit I grabbed.

I'll wait for your PR to be merged to master, then rebase. Thank you!

File: csp-mixin/panels/azurequeuestore.libsonnet (hunk 1 above)
Comment (rgeyer) [reply csp-mixin/panels/azurequeuestore.libsonnet]: Awww man.. Purple is so pretty tho ;)

I set it to `text` to be consistent.

**Pull request merged**